ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhoai-addon-interop__v4.15-production.yaml

```diff
@@ -0,0 +1,164 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  openshift-cli-installer:
+    name: openshift-cli-installer
+    namespace: ci
+    tag: latest
+  rhoai-tests:
+    name: ods-ci
+    namespace: ci
+    tag: stable
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: smoke-rosa-rhoai-addon-baseline-fips
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com;ocm-env=production
+      ADDONS_OPERATORS_RUN_IN_PARALLEL: "false"
+      API_HOST: production
+      CLUSTER_NAME_PREFIX: rhods
```

Comment:
Should the `CLUSTER_NAME_PREFIX: rhods` be changed to rhoai, since thats the new name we are going with ?

Reply:
yes it should, will change, thanks!